packages/gatsby-theme-newrelic/src/components/Link.js

```diff
@@ -60,6 +75,7 @@ const Link = ({ to, onClick, ...props }) => {
 Link.propTypes = {
   onClick: PropTypes.func,
   to: PropTypes.string.isRequired,
+  instrumentation: PropTypes.object),
```

Comment:
```suggestion
  instrumentation: PropTypes.object,
```

Looks like the build/tests are failing due to this syntax error 😄

Reply:
I really need to get some sleep or something 😦 sorry about that, very careless